src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy

```diff
@@ -292,6 +294,7 @@ class K8sServiceImpl implements K8sService {
                 .withActiveDeadlineSeconds( timeout.toSeconds() )
                 .withRestartPolicy("Never")
                 .addAllToVolumes(volumes)
+                .withOverhead(null)
```

Comment:
This looks useless

Reply:
This is added to avoid this error
`HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"pods \"build-090844c190db1409-1\" is forbidden: pod rejected: Pod Overhead set without corresponding RuntimeClass defined Overhead","reason":"Forbidden","details":{"name":"build-090844c190db1409-1","kind":"pods"},"code":403}`

Reply:
https://github.com/seqeralabs/wave/pull/553#issuecomment-2225937347

Reply:
Still not sure about this. If you look at the implementation of `withOverhead` you will fine 

```
    public <K, V> A withOverhead(Map<String, Quantity> overhead) {
        if (overhead == null) {
            this.overhead = null;
        } else {
            this.overhead = new LinkedHashMap(overhead);
        }

        return this;
    }
```

so, what's the value of using with `null`? also, i've tested in local k8s without it and I found no problem. Can you please double check?

Reply:
Not needed in my tests. Removed it [8d46fb7](https://github.com/seqeralabs/wave/pull/553/commits/8d46fb7d9e7391c54dfa610bedf381d0e235e256)